promgen/locale/ja/LC_MESSAGES/django.po

```diff
@@ -105,8 +105,8 @@ msgid "Sync Farm"
 msgstr "Farmを同期"
 
 #: templates/promgen/project_detail_hosts.html:58
-msgid "New Farm"
-msgstr "New Farm"
+msgid "Register Farm"
+msgstr "Register Farm"
```

Comment:
Since we're updating the others, we can also update this to match our hosts register

```suggestion
msgstr "Farmを登録"
```

Reply:
Done in 55e9d76.